src/components/Chart/Table.js

```diff
@@ -0,0 +1,233 @@
+          <tr>
+            {columns.map((tableHead, index) => (
+              <th
+                {...styles.header}
+                {...colorScheme.set('borderBottomColor', 'text')}
+                style={{
+                  textAlign: numberColumns.includes(tableHead)
+                    ? 'right'
+                    : 'left',
+                  cursor: 'pointer',
+                  whiteSpace: sortBy.key === tableHead ? 'nowrap' : undefined
+                }}
+                key={index}
+                onClick={() => setSort(columns[index])}
+              >
+                {tableHead}
+                {sortBy.key &&
+                  sortBy.key === tableHead &&
+                  (sortBy.order === 'desc' ? (
+                    <ExpandMoreIcon style={{ paddingLeft: '2px' }} />
+                  ) : (
+                    <ExpandLessIcon style={{ paddingLeft: '2px' }} />
+                  ))}
+              </th>
+            ))}
+          </tr>
+        </thead>
+        <tbody style={{ marginTop: '5px' }}>
+          {parsedData.map((row, rowIndex) => (
+            <tr
+              key={rowIndex}
+              {...(rowIndex % 2 !== 0 &&
+                colorScheme.set('backgroundColor', 'hover'))}
+            >
+              {columns.map((cellKey, cellIndex) => (
+                <Cell
+                  key={cellIndex}
+                  type={tableColumns.find(d => d.column === cellKey)?.type}
+                  width={tableColumns.find(d => d.column === cellKey)?.width}
+                  color={tableColumns.find(d => d.column === cellKey)?.color}
+                  value={row[cellKey]}
+                  colorScale={colorScale}
+                >
+                  {numberColumns.includes(cellKey)
+                    ? numberFormatter(row[cellKey])
+                    : row[cellKey]}
+                </Cell>
+              ))}
+            </tr>
+          ))}
+        </tbody>
+      </table>
+    </div>
+  )
+}
+
+export const propTypes = {
+  values: PropTypes.array.isRequired,
+  tableColumns: PropTypes.arrayOf(
+    PropTypes.shape({
+      column: PropTypes.string,
+      type: PropTypes.string,
+      width: PropTypes.string,
+      color: PropTypes.bool
+    })
+  ),
+  numberFormat: PropTypes.string.isRequired,
+  defaultSortColumn: PropTypes.string,
+  colorMap: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
+  colorRange: PropTypes.oneOfType([PropTypes.string, PropTypes.array]),
+  colorRanges: PropTypes.shape({
+    diverging2: PropTypes.array.isRequired,
+    sequential3: PropTypes.array.isRequired,
+    discrete: PropTypes.array.isRequired
+  }).isRequired
+}
+
+Table.defaultProps = defaultProps.Table
+
+Table.propTypes = propTypes
+
+export default Table
+
+const Cell = props => {
+  const { type, width, color, colorScale, value, children } = props
+  return (
+    <td
+      {...(type === 'number' && styles.cellNumber)}
+      {...styles.cell}
+      style={{
+        width: +width,
```

Comment:
OOOps two more things:

<img width="1245" alt="Screenshot 2022-01-21 at 16 24 40" src="https://user-images.githubusercontent.com/410211/150553170-fd25a67b-a0fb-4f0b-8f5c-24115c8ba349.png">


```suggestion
        width: width !== undefined ? +width : undefined,
```